docs/source/reference/config.rst

```diff
@@ -22,4 +22,38 @@ Access configuration values
   :toctree: api/
 
   conf
+  config.get_cache_dir
   config.get_config_dir
+
+
+Default Cache Directory Migration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting from ``Lightkurve`` version ``2.3.0``, the default cache directory is
```

Comment:
@barentsen I intend to merge this PR in the next few days and bump the version to v2.4.0 . 
The version needs to be synced up with the one specified in the migration doc above.

Let me know if there is any concern.